go-controller/pkg/ovn/service.go

```diff
@@ -385,6 +397,20 @@ func (ovn *Controller) createService(service *kapi.Service) error {
 					klog.Infof("Service Reject ACL created for ClusterIP service: %s, namespace: %s, via: "+
 						"%s:%s:%d, ACL UUID: %s", service.Name, service.Namespace, svcPort.Protocol,
 						service.Spec.ClusterIP, svcPort.Port, aclUUID)
+					// Cloud load balancers: directly reject traffic from pods
+					for _, ing := range service.Status.LoadBalancer.Ingress {
+						if ing.IP == "" {
+							continue
+						}
+						aclUUID, err := ovn.createLoadBalancerRejectACL(loadBalancer, ing.IP, svcPort.Port, svcPort.Protocol)
+						if err != nil {
+							klog.Errorf("Failed to create reject ACL for Ingress IP: %s, load balancer: %s, error: %v",
+								ing.IP, loadBalancer, err)
+						} else {
+							klog.Infof("Reject ACL created for Ingress IP: %s, load balancer: %s, %s", ing.IP,
+								loadBalancer, aclUUID)
+						}
```

Comment:
likewise it seems like the reject-acl-creation code in `createService` and `deleteEndpoints` could share more / duplicate less

Reply:
yeah I noticed that as well. I'm hoping we get rid of all of the ACL reject code soon with:
https://bugzilla.redhat.com/show_bug.cgi?id=1883957